src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -266,7 +280,7 @@ public static boolean isSuppressed(AuditEvent event) {
         for (Entry entry : entries) {
             final boolean afterStart = isSuppressedAfterEventStart(line, column, entry);
             final boolean beforeEnd = isSuppressedBeforeEventEnd(line, column, entry);
-            final String checkName = entry.getCheckName();
+            final String checkName = setCheckName(entry.getCheckName());
```

Comment:
Do not change this line, create new line that takes check name and remote suffix "check" and do extra `||` in expression below